### PR TITLE
Add Static IP Script

### PR DESCRIPTION
**Pull request by MRDT-Software**

### CODE REVIEW

**Review by Byrdman32 — APPROVED** [review]

File: tools/set_static_ip.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+	echo "# interfaces(5) file used by ifup(8) and ifdown(8)" > "${NETWORK}"
+	echo "# Include files from /etc/network/interfaces.d:" >> "${NETWORK}"
+	echo "source-directory /etc/network/interface" >> "${NETWORK}"
+	echo "" >> "${NETWORK}"
+	echo "# Set static ip on ethernet interface of 192.168.1.139" >> "${NETWORK}"
+	echo "allow-hotplug eth0" >> "${NETWORK}"
+	echo "iface eth0 inet static" >> "${NETWORK}"
+	echo "        address 192.168.1.139" >> "${NETWORK}"
+	echo "        netmask 255.255.255.0" >> "${NETWORK}"
+	echo "        gateway 192.168.0.1" >> "${NETWORK}"
```
Comment (ClayJay3) [inline tools/set_static_ip.sh]: Default gateway is not on the same subnet. Host won't be able to talk outside of subnet.

File: tools/set_static_ip.sh [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        echo "# interfaces(5) file used by ifup(8) and ifdown(8)" > "${NETWORK}"
+        echo "# Include files from /etc/network/interfaces.d:" >> "${NETWORK}"
+        echo "source-directory /etc/network/interface" >> "${NETWORK}"
+        echo "" >> "${NETWORK}"
+        echo "# Set static ip on ethernet interface of 192.168.1.139" >> "${NETWORK}"
+        echo "#allow-hotplug eth0" >> "${NETWORK}"
+        echo "#iface eth0 inet static" >> "${NETWORK}"
+        echo "#        address 192.168.1.139" >> "${NETWORK}"
+        echo "#        netmask 255.255.255.0" >> "${NETWORK}"
+        echo "#        gateway 192.168.0.1" >> "${NETWORK}"
```
Comment (ClayJay3) [inline tools/set_static_ip.sh]: Same thing here. Should be 192.168.3.1

File: tools/set_static_ip.sh [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+	echo "Turning On - Static IP"
+
+	echo "# interfaces(5) file used by ifup(8) and ifdown(8)" > "${NETWORK}"
+	echo "# Include files from /etc/network/interfaces.d:" >> "${NETWORK}"
+	echo "source-directory /etc/network/interface" >> "${NETWORK}"
+	echo "" >> "${NETWORK}"
+	echo "# Set static ip on ethernet interface of 192.168.1.139" >> "${NETWORK}"
+	echo "allow-hotplug eth0" >> "${NETWORK}"
+	echo "iface eth0 inet static" >> "${NETWORK}"
+	echo "        address 192.168.1.139" >> "${NETWORK}"
```
Comment (ClayJay3) [inline tools/set_static_ip.sh]: Jetson's new IP is on VLAN 3: 192.168.3.100

File: tools/set_static_ip.sh [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+	echo "Turning Off -  Static IP"
+
+        echo "# interfaces(5) file used by ifup(8) and ifdown(8)" > "${NETWORK}"
+        echo "# Include files from /etc/network/interfaces.d:" >> "${NETWORK}"
+        echo "source-directory /etc/network/interface" >> "${NETWORK}"
+        echo "" >> "${NETWORK}"
+        echo "# Set static ip on ethernet interface of 192.168.1.139" >> "${NETWORK}"
+        echo "#allow-hotplug eth0" >> "${NETWORK}"
+        echo "#iface eth0 inet static" >> "${NETWORK}"
+        echo "#        address 192.168.1.139" >> "${NETWORK}"
```
Comment (ClayJay3) [inline tools/set_static_ip.sh]: Jetson's new IP is on VLAN 3: 192.168.3.100

**Review by ClayJay3 — CHANGES_REQUESTED** [review]

File: tools/set_static_ip.sh (hunk 1 above)
Comment (MRDT-Software) [reply tools/set_static_ip.sh]: Should this be `192.168.3.1`

Comment (ClayJay3) [reply tools/set_static_ip.sh]: Correct.

File: tools/set_static_ip.sh [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+NETWORK=/etc/network/interfaces
+
+if [ $1 = "on" ]; then
+	echo "Turning On - Static IP"
+
+	echo "# interfaces(5) file used by ifup(8) and ifdown(8)" > "${NETWORK}"
+	echo "# Include files from /etc/network/interfaces.d:" >> "${NETWORK}"
+	echo "source-directory /etc/network/interface" >> "${NETWORK}"
+	echo "" >> "${NETWORK}"
+	echo "# Set static ip on ethernet interface of 192.168.1.139" >> "${NETWORK}"
```
Comment (ClayJay3) [inline tools/set_static_ip.sh]: Change to match address 192.168.3.100 to avoid confusion later.

File: tools/set_static_ip.sh [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+
+	echo "Static IP has been turned ON"
+elif [ $1 = "off" ]; then
+	echo "Turning Off -  Static IP"
+
+        echo "# interfaces(5) file used by ifup(8) and ifdown(8)" > "${NETWORK}"
+        echo "# Include files from /etc/network/interfaces.d:" >> "${NETWORK}"
+        echo "source-directory /etc/network/interface" >> "${NETWORK}"
+        echo "" >> "${NETWORK}"
+        echo "# Set static ip on ethernet interface of 192.168.1.139" >> "${NETWORK}"
```
Comment (ClayJay3) [inline tools/set_static_ip.sh]: Change to match address 192.168.3.100 to avoid confusion later.

**Review by ClayJay3 — APPROVED** [review]

**Pull request merged**